packages/terra-functional-testing/src/config/wdio.conf.js

```diff
@@ -165,4 +166,17 @@ exports.config = {
     ui: 'bdd',
     timeout: 60000,
   },
+
+  /* Terra defined viewport sizes. */
+  terraViewports: {
+    tiny: { width: 470, height: 768, name: 'tiny' },
+    small: { width: 622, height: 768, name: 'small' },
+    medium: { width: 838, height: 768, name: 'medium' },
+    large: { width: 1000, height: 768, name: 'large' },
+    huge: { width: 1300, height: 768, name: 'huge' },
+    enormous: { width: 1500, height: 768, name: 'enormous' },
+  },
```

Comment:
With these being static it might be worth discussing if these should be defaulted within the service and excluded from being defined in this top level configuration

Reply:
Would these be changed for mpages tests? Do we know if anyone ever changes these? I lean towards excluding this from config.

Reply:
These won't change in MPages

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/438/commits/3db345e69d9b12382795557acce0285b0bd8abbc